tsconfig.json

```diff
@@ -1,6 +1,5 @@
 {
   "compilerOptions": {
-    "esModuleInterop": true,
```

Comment:
it's best *not* to use this option when building libraries - it might "taint" the dependency and force all of the consumers to also use this option

Reply:
This particular package wasn't actually prone to this, so it's fine right now.